examples/one-2-one/test/acceptance/test-helper.ts

```diff
@@ -0,0 +1,24 @@
+import {One2oneApplication} from '../..';
+import {
+  createRestAppClient,
+  givenHttpServerConfig,
+  Client,
+} from '@loopback/testlab';
+
+export async function setupApplication(): Promise<AppWithClient> {
+  const app = new One2oneApplication({
+    rest: givenHttpServerConfig(),
+  });
+
+  await app.boot();
+  await app.start();
+
+  const client = createRestAppClient(app);
+
+  return {app, client};
+}
+
+export interface AppWithClient {
+  app: One2oneApplication;
+  client: Client;
+}
```

Comment:
Please add acceptance-level tests for each controller you are adding to this example application:
 - `SupplierController`
 - `SupplierAccountController`
 - `AccountController` (if you decide to keep it)